crunch/crunch.c

```diff
@@ -209,7 +209,9 @@ uint8_t tryRegistration(void *testSuite)
 		dlclose(testSuite);
 		return FALSE;
 	}
+	testPrintf("Calling registerTests (%p)\n", registerTests);
```

Comment:
Given this and the other testPrintf() were intended for debugging and would become part of the permanent output of the program this way, we suggest dropping these two changes